cmd/logs.js

```diff
@@ -28,14 +29,25 @@ function validateTimestamp(ts) {
   return moment(ts, moment.ISO_8601, true).isValid();
 }
 
-function logs(from, to, command, cb) {
+function validateNumber(number) {
+  if (number == null) return true;
+  if (number === 0 || number === '0') return false;
```

Comment:
Maybe, func name could be changed to something a bit more descriptive or a comment be added? Like isValidInteger, isValidNonZeroInteger...